### ISSUE TITLE
Build with provided Dockerfile fails

### ISSUE THREAD

**Issue by Mxttak** [Kitware/TeleSculptor#410]
When building with provided Dockerfile, the following error occurs:

```
/usr/bin/ld: CMakeFiles/cuda_compile_1.dir/cuda_compile_1_generated_integrate_depth_maps.cu.o: relocation R_X86_64_PC32 against symbol `_ZTVSt9basic_iosIcSt11char_traitsIcEE@@GLIBCXX_3.4' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: Bad value
```

**Comment by mleotta** [comment]
Can you report which branch you are trying to build?  Are you building `release` or `master`?

We fixed a similar issue a few months ago in KWIVER: Kitware/kwiver#1098.  If you are building `release` that fix might not have been applied here.  If that's the case we can update the KWIVER used by the `release` branch.

**Comment by Mxttak** [comment]
Thanks for the super quick reply!

Sorry I didn't mention it. It has been ```release```. I changed it to ```master``` and started a new build. I will report the result asap

**Comment by Mxttak** [comment]
Hi Matt,

you were right, building on ```master``` worked. Thanks for the hint!

However, I had to update cmake in the image via 
```RUN wget --no-check-certificate -O cmake.sh https://github.com/Kitware/CMake/releases/download/v3.15.5/cmake-3.15.5-Linux-x86_64.sh && sh ./cmake.sh --prefix=/usr/local --skip-license```

Shall I create a PR?

Best,
Maxim

**Comment by mleotta** [comment]
Yes, please do make a PR for this fix in `master`.  Thanks!

Also, I just merged #411 which I think will fix the build for `release` too (without requiring the CMake upgrade)

**Comment by Mxttak** [comment]
Thanks. I was able to build the image without a cmake upgrade.
